mscore/scoreview.cpp

```diff
@@ -26,6 +26,7 @@
 #include "navigator.h"
 #include "preferences.h"
 #include "scoreaccessibility.h"
+#include "scoretreeview.h"
```

Comment:
Why include (and setup) your dock widget in `scoreview.cpp` rather than `musescore.cpp`? The dock widget is not part of the ScoreView anymore; it is a separate widget like the Inspector, Palettes, or Mixer.

Reply:
I moved ScoreTreeView to `musescore.cpp` and added the action "Show Tree Debugger" to the debug menu so that it's shown only in debug builds. Also renamed it and made it inherit from QDockWidget.